### ISSUE TITLE
Unable to reproduce the experimental results from the paper

### ISSUE THREAD

**Issue by wangziyue-nubot** [minwoo0611/HeLiOS#12]
I downloaded the preprocessed data you uploaded to Google Cloud, including four types of LiDAR for DCC456, KAIST456, and Riverside456, a total of 36 sequences, as well as your overlap_matrix_training.txt for training. Then, according to your settings, using Roundabout01 Ouster as the database set, I downloaded your overlap_matrix_Soundabout.exe and the other 11 sequences of Roundabout as the query set. The specific results are as follows, which are about 10% weaker than the results in your paper. The device I am using is RTX 4070Ti, which has been modified in the config file. batch_split_size = 32 
I look forward to your help in training a network with similar performance to your open-source pre trained network.

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/a725e6b2-3a84-4d30-804f-81c942fdf79f" />

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/1c5a0cd1-b6c4-4f1e-bf46-6a9c41f866aa" />

**Comment by minwoo0611** [comment]
Hi @wangziyue-nubot,

Thank you for sharing the details. I think it is strange that the performance differs by about 10%, as that’s quite a significant gap.

I recommend first testing with our released pre-trained model to verify whether its performance is consistent with the results reported in our paper. That way, we can confirm whether the issue comes from the training process or from the evaluation settings.

In the meantime, I will also try to retrain our model to check its reproducibility on my side. However, please note that there may be some delay since I am currently preparing a paper submission.

Anyway, thank you again for your interest in our work!

**Comment by wangziyue-nubot** [comment]
Thank you for your reply. 

Prior to this, I have tested using your open pre trained model and the results are consistent with those in the paper.

I look forward to your reply

**Comment by minwoo0611** [comment]
Hi @wangziyue-nubot,

I found that the problem is from overfitting. In the default config, it almost converges to the best performance in 2-30 epochs, and it decreases from that epoch.

So, I recommend you decrease the learning rate (5e-4 or 1e-4) or make a validation pickle to deviate the overfitting using other sequences (Town or Bridge). I checked that in these settings, I got similar performances, which are represented in the paper.

I'll also commit the new configuration file, which reflects this, ASAP.

If you have further inquiries, please make an issue.

Thank you.

**Comment by wangziyue-nubot** [comment]
I made new attempts, including training with DCC456, KAIST456, and Riverside456, validating with Bridge14, and testing with Roundabout1_Ouster as the database and Roundabout123 as the test set. Simultaneously attempted learning rates of 5e-4 and 1e-4 respectively. The data is sourced from Google Cloud Drive. I am pleased that the newly trained HeliOS-S model has improved in performance compared to the previous one, but there is still a gap of about 5% compared to the results in the paper. Looking forward to your new configuration file.

<img width="2560" height="1440" alt="Image" src="https://github.com/user-attachments/assets/87be5535-e78d-446e-8d6a-6e0992dcd395" />

Results under a learning rate of 1e-4.

<img width="2430" height="1307" alt="Image" src="https://github.com/user-attachments/assets/326e7eb9-1579-4435-aeaa-7dc5bcee4353" />

Results under a learning rate of 5e-4.

**Comment by minwoo0611** [comment]
When I tried to train using this setup, I got the below results.

```
num_workers = 4
batch_size = 2048
batch_split_size = 200
val_batch_size = 256

lr = 5e-4
epochs = 150
scheduler_milestones = 40, 100
aug_mode = 1
weight_decay = 1e-4
```

Due to format differences, it looks differing from your result, but they are the results from 0-0 to 0 -11.
I'll retrain helios-s, with changing batch_split_size from 200 to 32, and share the results in here.
But I think it is really weird.

<img width="248" height="649" alt="Image" src="https://github.com/user-attachments/assets/29cf2425-5466-4a72-8eb3-29ceaf4e1dec" />

**Comment by wangziyue-nubot** [comment]
Thank you for your help and guidance. The training results have greatly improved compared to before.
